### PR TITLE
Add beta.5 changelog

### PR DESCRIPTION
**Pull request by joeyaiello**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,105 @@
...
+
+### Build/test and code cleanup
+
+* Add build check for MFC for Visual C++ during Windows builds.
+  This fixes a long-standing (and very frustrating!) issue with missing build dependencies! (#4185) (Thanks to @KirkMunro!)
+* Add tests for built-in type accelerators. (#4230) (Thanks to @dchristian3188!)
+* Increase code coverage of `Get-ChildItem` on file system. (#4342) (Thanks to @jeffbi!)
+* Increase test coverage for `Rename-Item` and `Move-Item`. (#4329) (Thanks to @jeffbi!)
+* Add test coverage for Registry provider. (#4354) (Thanks to @jeffbi!)
+* Fix C# tests for `ValidateSetValuesGenerator`. (#4253) (Thanks to @iSazonov!)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: This one can be removed, it's actually the same as the `ValidateSetAttribute` enhancement work we mentioned above.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,105 @@
...
+* Create script to install latest PowerShell from Microsoft package repositories (or Homebrew) on non-Windows platforms. (#3608) (Thanks to @DarwinJS!)
+* Enable MSI upgrades rather than a side-by-side install. (#4259)
+* Add a checkbox to open PowerShell after the Windows MSI installer has finished. (#4203) (Thanks to @bergmeister!)
+* Add Amazon Linux compatibility to `install-powershell.sh`. (#4360) (Thanks to @DarwinJS!)
+* Add ability to package all of PowerShell Core as a NuGet package. (#4363)
+
+### Build/test and code cleanup
+
+* Add build check for MFC for Visual C++ during Windows builds.
+  This fixes a long-standing (and very frustrating!) issue with missing build dependencies! (#4185) (Thanks to @KirkMunro!)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: We should add one more item here:
#4335 - Separate Windows PSRP binary build out of Start-PSBuild
Starting from beta.5, `Start-PSBuild` doesn't build PSRP binary on windows. Instead, the PSRP binary is consumed from a NuGet package.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,105 @@
...
+
+### General cmdlet updates and fixes
+
+* Add `-SkipHeaderValidation` switch to `Invoke-WebRequest` and `Invoke-RestMethod` to support adding headers without validating the header value. (#4085)
+* Add support for `Invoke-Item -Path <folder>`. (#4262)
+* Fix `ConvertTo-Html` output when using a single column header. (#4276)
+* Fix output of `Length` for `FileInfo` when using `Format-List`. (#4437)
+* Fix an issue in implicit remoting where restricted sessions couldn't use `Get-FormatData –PowerShellVersion`. (#4222)
+* Fix an issue where `Register-PSSessionConfiguration` fails if `SessionConfig` folder doesn't exist. (#4271)
+* Remove the `*-Counter` cmdlets in `Microsoft.PowerShell.Diagnostics` due to the use of unsupported APIs until a better solution is found. (#4303)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: I thought we were going to highlight breaking changes?  Same for LocalAccounts

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,105 @@
...
+* Fix regressions that cause implicit remoting tests to fail. (#4326)
+* Disable legacy UTC and SQM Windows telemetry by enclosing the code in '#if LEGACYTELEMETRY'. (#4190)
+
+#### Cleanup `#if CORE` code
+
+PowerShell 6.0 will be exclusively built on top of CoreCLR,
+so we are removing a large amount of code that's built only for FullCLR.
+To read more about this, check out [this blog post](https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/).
+
+* Cleanup `#if CORE` in `ManagedEntrance.cs` (#4373) (Thanks to @iSazonov!)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: We should combine these per contributor call out rather than list them all out.  It's not useful to end users.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
I'm taking over this change and will submit a new PR. Close this PR.

**Pull request closed without merging**